### ISSUE TITLE
Servicegroups

### ISSUE THREAD

**Issue by etemam** [shinken-monitoring/mod-webui#392]
Hi,

Fist of all thanks for the job. Your ui is a great work. 

I have a problem with servicegroups. None of the services appears in the groups. The services as declared like this one:

```
define service{
    use                           critic-service
    servicegroups           MailServices
    host_name                MYHOSTS
    service_description   Number of Imap Login
    check_command       check_nrpe!check_nb_imap_login
}
```
The service does not appear in MailServices defined by:
```
define servicegroup{
    servicegroup_name       MailServices
    alias                   All mail services
}
```
It is the case for all services (while hostgroups is working perfectly).

Thanks!

**Comment by mohierf** [comment]
You are right ... the groups appear correctly on the service view, the service group appears correctly n the services groups list ... but is is empty!

**Comment by mohierf** [comment]
I made a test and the services groups are correctly managed when you declare the `servicegroups` property in a service template rather than directly in the service definition ...

```
define servicegroup{
   servicegroup_name    web
   alias                All http services
}

define service{
   service_description  Http
   use                  generic-service
   register             0
   host_name            http
   check_command        check_http
   servicegroups        web
}
```

It is a bug in the Shinken configuration parser, not in the WebUI ... I will open an issue in Shinken project for this.

**Comment by etemam** [comment]
Hi,

thanks for reply. I do not know if it is a bug of shinken or webui but:
The bug appear with shinken 2.4.2, webui
The bug does not appear with shinken 2.0.3 and thruk

Sincerly

**Comment by mohierf** [comment]
The fix for #423 may fix this problem

**Comment by etemam** [comment]
Indeed, problem is solved.
